src/styles/card-content.scss

```diff
@@ -0,0 +1,111 @@
+$main-text-color: #060b35;
+$secondary-text-color: #616070;
+
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+body {
+  width: 200px;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 17px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    flex-shrink: 0;
+    margin: 16px 0 40px;
+  }
+}
+
+.card-content {
```

Comment:
it seems like you have 2 BEM-blocks here: `card` and `card-content`. There should be only one (`card`). You may just get rid of the `card-content` wrapper and make all its children `card` block's children